### ISSUE TITLE
Datepicker: First day of month gets gray highlight upon opening

### ISSUE THREAD

**Issue by sergeu90** [ng-bootstrap/ng-bootstrap#2000]
Hellow everyone.Can you help me.How can i remove first day of month gets gray highlight upon opening?
![image](https://user-images.githubusercontent.com/7217517/33368428-b94ef06e-d502-11e7-9743-b710e632aaca.png)

**Comment by pkozlowski-opensource** [comment]
@maxokorokov if I recall properly this is to "highlight focus" and there was a prior discussion about it. Do you recall the details?

**Comment by maxokorokov** [comment]
Yes, that's the "keyboard focus", when you focus the datepicker, press on keyboard arrow keys and you'll see it move.

We're setting `btn-light active`, so you should be able to access the css by changing something like:
```css
.ngb-dp-day > .active {
}
/* or for non-currently selected date*/
.ngb-dp-day > .active:not(.bg-primary) {
}

```

**Comment by umerali523** [comment]
@maxkorz  @pkozlowski-opensource i was using @ngbootstrap 2.0.0. In datepicker there was an issue of autoclosing Datepicker on outside click which resolved in 3.0.0 but as i moved to the current version i saw this #2000 issue.  If it was resolved in 2.0.0 why it appeared again in current version? Can you help me out?
